packages/react/src/patterns/sub-patterns/PictogramItem/README.md

```diff
@@ -54,16 +60,23 @@ ReactDOM.render(<App />, document.querySelector('#app'));
 | ----------- | -------- | --------- | ------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------- |
 | `heading`   | YES      | String    | null          | Heading text                                                                                                                                            |
 | `copy`      | YES      | String    | null          | Copy text                                                                                                                                               |
-| `Pictogram` | YES      | Component | null          | Pictogram component imported from `@carbon/pictograms-react` package                                                                                    |
+| `Pictogram` | YES      | Object    | null          | See `Pictogram` bellow. package                                                                                                                         |
 | `cta`       | NO       | Object    | null          | Object with CTA data, check [CTA](https://github.com/carbon-design-system/ibm-dotcom-library/tree/master/packages/react/src/components/CTA/README.md)👀 |
 
+## Pictogram
+
+| Name | Required | Data Type | Default Value | Description | | `src` | YES |
+Component | null | Pictogram component imported from `@carbon/pictograms-react`
+| | `ariaLabel` | NO | string | null | Aria-label for the Pictogram | |
+`ariaHidden` | NO | string | null | Aria-hidden for the Pictogram |
```

Comment:
I have mixed feelings about this. If we start listing these all out here we begin to start listing out the whole HTML attribute options. Maybe just add a section for accessibility instead and show how by adding an aria label you can make this accessible for screen readers?